www/front_src/src/Resources/Filter/Criterias/index.tsx

```diff
@@ -31,15 +33,36 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+const getCriterias = (filterWithParsedSearch): Array<Criteria> => {
+  const getSelectableCriteriaByName = (name: string): CriteriaDisplayProps =>
+    selectableCriterias[name];
+
+  const isNonSelectableCriteria = (criteria: Criteria): boolean =>
+    pipe(({ name }) => name, getSelectableCriteriaByName, isNil)(criteria);
+
+  const criterias = sortBy<Criteria>(
+    ({ name }) => criteriaNameSortOrder[name],
+    filterWithParsedSearch.criterias,
+  );
+
+  return pipe(
+    reject(isNonSelectableCriteria) as (criterias) => Array<Criteria>,
+  )(criterias);
+};
+
```

Comment:
I guess you need to wait for the other PR

Reply:
ok